index.js

```diff
@@ -32,16 +33,28 @@ module.exports = function standardVersion (argv, done) {
     } else {
       checkpoint(args, 'skip version bump on first release', [], chalk.red(figures.cross))
     }
-
-    outputChangelog(args, function (err) {
+    runLifecycleHook(args, 'post-bump', newVersion, hooks, function (err) {
       if (err) {
+        printError(args, err.message)
         return done(err)
       }
-      commit(args, newVersion, function (err) {
+
+      outputChangelog(args, function (err) {
         if (err) {
           return done(err)
         }
-        return tag(newVersion, pkg.private, args, done)
+        runLifecycleHook(args, 'pre-commit', newVersion, hooks, function (err) {
+          if (err) {
```

Comment:
Just a suggestion here, as this would help my own use case which I will outline below. If you allow the lifecycle hook to mutate/return a modified args struct that could allow more flexibility. (I'm currently considering using a `prepare-commit-message` hook to achieve this).

My company uses PivotalTracker which uses `[(Fixes|Finishes) #12312]` to indicate a story is fixed in a commit and then '[Delivers #12312]` to mark it as delivered and a github integration to detect the commit message and update the stories. What I'd want to do in this feature is change the `message` argument depending on all "fixed" commits since last version. (It'd be awesome if standard-version/conventional-changelog could pass all the commits it builds the changelog off to the hook as well but that might be asking for too much).

### Example
Git History:
```
fix: Some Issue [Fixes #123]
feat: Shiny Feature [Finishes #333]
chore(release): v1.1.2
```
On running `standard-version` I ideally want the release commit to be as follows so my stories automatically get updated to the delivered (for QA) state.
```
chore(release): v1.2.0

[Delivers #123]
[Delivers #333]
```
Assuming the hook allowed us to return a modified args I could append the new messages onto the `message` argument.